server/api/controllers/user.js

```diff
@@ -307,6 +309,54 @@ const loginPost = (req, res) => {
 				throw new Error(INVALID_CREDENTIALS + message);
 			}
 
+			const lastLogins = await toolsLib.user.findUserLastLogins(user, true);
+			let suspiciousLogin = false;
+			if (!referer && !domain) {
+				suspiciousLogin = true;
```

Comment:
As we checked, mobile app use cases do not have referred and domain, so this logic would always end up banning the app users. We should drop this line of code @fetok12